README.rst

```diff
@@ -93,6 +93,31 @@ test_publish_article.py:
         assert article.is_published
 
 
+Less code for scenario test function
+------------------------------------
+
+pytest-bdd `scenario` decorator can manage the scenario function name for you automatically to reduce the amount of
+code you need to write:
+
+.. code-block:: python
+
+    from pytest_bdd import scenario
+
+    @scenario('publish_article.feature', 'Publishing the article')
+    def _():
+        pass
+
+As a result, module will get the function:
+
+
+.. code-block:: python
+
+
+    def test_publishing_the_article():
```

Comment:
So you won't see the "test_publishing_the_article" explicitly? How do you copy-paste it to run py.test -k test_publishing_the_article then?